doc/biocpp/presentation/index.html

```diff
@@ -135,39 +135,39 @@ <h3>Schedule<small>*</small>:</h3>
 You have three options to install C++:
 
 * <!-- .element: class="fragment" -->
-  Use your web-browser.
-  * Recommended for <i class="fa fa-windows" aria-hidden="true"></i> Windows or
+  Use your web browser.
+  * Recommended for <i class="fa fa-windows" aria-hidden="true"></i> Windows and
   * people who want to keep a clean system.
 * <!-- .element: class="fragment" -->
   Install C++ on your system.
-  * Recommended for <i class="fa fa-apple" aria-hidden="true"></i> MacOS or
+  * Recommended for <i class="fa fa-apple" aria-hidden="true"></i> MacOS and
     <i class="fa fa-linux" aria-hidden="true"></i> Linux
 * <!-- .element: class="fragment" -->
   Use a VM.
-  * Recommended for <i class="fa fa-windows" aria-hidden="true"></i> Windows or
+  * Recommended for <i class="fa fa-windows" aria-hidden="true"></i> Windows and
   * people who want to keep a clean system.
 </textarea>
 </section>
 
 
 <section data-markdown>
 <textarea data-template>
-###### Setup C++: Use your web-browser (I)
+###### Setup C++: Use your web browser (I)
 
 <img src="img/gitpod.io.png" height="420" class="fragment">
 </textarea>
 </section>
 
 <section data-markdown>
 <textarea data-template>
-###### Setup C++: Use your web-browser (I)
+###### Setup C++: Use your web browser (I)
 
 * <!-- .element: class="fragment" -->
   Go to https://gitpod.io/#https://github.com/seqan/learning-resources
 * <!-- .element: class="fragment" -->
-  It is 50h/month for free, plenty of time for this tutorial.
+  It is free for 50h/month, more than enough time for this tutorial.
 * <!-- .element: class="fragment" -->
-  You need a github account. Create one if you don't have one.
+  You need a GitHub account. Create one if you don't have one.
```

Comment:
```suggestion
  You need a [GitHub](https://github.com/) account. Create one if you don't have one.
```

Edit: Do links work at all here in the slides?

Reply:
Yes, they do